site/src/client.tsx

```diff
@@ -38,16 +38,16 @@ const ScrollManager = ({ children }: { children: ReactNode }) => {
 export default (app: RenderContext) => {
   initUpdates(new Date(app.renderDate), app.versionMap, app.currentVersion);
 
-  hydrate(
-    <HeadProvider>
+  hydrateRoot(
+    document.getElementById('app')!,
```

Comment:
Is this `!` a real thing, or typo?

Reply:
Haha, nah i got this from their [migration guide](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis). I originally added a null check on the `getElementById` call, but then spotted this in their docs.

Reply:
Oh the null thing, forgot about types 😅